libraries/core/src/main/kotlin/com/bumble/appyx/core/modality/BuildContext.kt

```diff
@@ -22,6 +26,18 @@ data class BuildContext(
             )
     }
 
-    fun <T : NodeCustomisation> getOrDefault(defaultCustomisation: T) : T =
+    val identifier: String by lazy {
```

Comment:
Make this internal?

Reply:
Add a comment at the very least saying to not use it, Incubating?

Reply:
We cannot make this internal unfortunately. Not sure if I should mark this with a comment as this will be part of the 1.0 API